Sources/Contour/Configuration/BusConfiguration.cs

```diff
@@ -158,6 +160,9 @@ public IEnumerable<ISenderConfiguration> SenderConfigurations
         /// </summary>
         public IPayloadConverter Serializer { get; private set; }
 
+        /// <inheritdoc />
+        public IEnumerable<string> ExcludedIncomingHeaders { get; private set; }
```

Comment:
What is the configuration scope of this setting?
If it is Endpoint then it needs to be moved to EndpointOptions etc.